packages/core/CHANGELOG.md

```diff
@@ -1083,7 +1083,7 @@ Nothing. Was published by mistake and does not contain valid code. Same as 0.0.1
 - Doc: Link to [Waku v2 Toy Chat specs](https://rfc.vac.dev/spec/22/) in README.
 - Examples (web chat): Persist nick.
 - Support for custom PubSub Topics to `Waku`, `WakuRelay`, `WakuStore` and `WakuLightPush`;
-  Passing a PubSub Topic is optional and still defaults to `/waku/2/default-waku/proto`;
+  Passing a PubSub Topic is optional and still defaults to `/waku/2/rs/0/0`;
```

Comment:
This change is not needed as it is change log file.

Reply:
Oooh true, apologies, didn't notice :)